.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish to PyPI
+on:
+  release:
+    types: [created]
```

Comment:
Although I am not entirely sure the difference betwewen `created` and `published`, let's stick with the [example](https://github.com/actions/starter-workflows/blob/8c4fc57de83ba017cd877a7fad3f4dd665c5c4b3/ci/python-publish.yml#L8) for now